runners/katacoda/templates/dockerCompose.md

```diff
@@ -0,0 +1,21 @@
+<%= text; %>
+
+## Docker Compose
+
+<%= cdCommand; %>
+
+<% if(!interrupt){ %>
+Now run docker-compose.
+<% } else { %>
+The docker container is already running.
```

Comment:
Here we can additionally mention that docker is stopped and restarted automatically by clicking on the command